%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% baposter Landscape Poster
% LaTeX Template
% Version 1.0 (11/06/13)
%
% baposter Class Created by:
% Brian Amberg (baposter@brian-amberg.de)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
% PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[landscape,a0paper,fontscale=0.285]{baposter} % Adjust the font scale/size here

\usepackage{graphicx} % Required for including images
\graphicspath{{figs/}} % Directory in which figures are stored

\usepackage{amsmath} % For typesetting math
\usepackage{amssymb} % Adds new symbols to be used in math mode

\usepackage{booktabs} % Top and bottom rules for tables
\usepackage{enumitem} % Used to reduce itemize/enumerate spacing
\usepackage{palatino} % Use the Palatino font
\usepackage[font=small,labelfont=bf]{caption} % Required for specifying captions to tables and figures

\usepackage{multicol} % Required for multiple columns
\setlength{\columnsep}{1.5em} % Slightly increase the space between columns
\setlength{\columnseprule}{0mm} % No horizontal rule between columns

\newcommand{\compresslist}{ % Define a command to reduce spacing within itemize enumerate environments, this is used right after \begin{itemize} or \begin{enumerate}
\setlength{\itemsep}{1pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
}

\definecolor{lightblue}{rgb}{0.145,0.6666,1} % Defines the color used for content box headers

\begin{document}

\begin{poster}
{
headerborder=closed, % Adds a border around the header of content boxes
colspacing=1em, % Column spacing
bgColorOne=white, % Background color for the gradient on the left side of the poster
bgColorTwo=white, % Background color for the gradient on the right side of the poster
borderColor=lightblue, % Border color
headerColorOne=black, % Background color for the header in the content boxes (left side)
headerColorTwo=lightblue, % Background color for the header in the content boxes (right side)
headerFontColor=white, % Text color for the header text in the content boxes
boxColorOne=white, % Background color of the content boxes
textborder=roundedleft, % Format of the border around content boxes, can be: none, bars, coils, triangles, rectangle, rounded, roundedsmall, roundedright or faded
eyecatcher=true, % Set to false for ignoring the left logo in the title and move the title left
headerheight=0.1\textheight, % Height of the header
headershape=roundedright, % Specify the rounded corner in the content box headers, can be: rectangle, small-rounded, roundedright, roundedleft or rounded
headerfont=\Large\bf\textsc, % Large, bold and sans serif font in the headers of content boxes
%textfont={\setlength{\parindent}{1.5em}}, % Uncomment for paragraph indentation
linewidth=2pt % Width of the border lines around content boxes
}
%-------------------------------------------------------------------------------
% TITLE SECTION
%-------------------------------------------------------------------------------
%
{\includegraphics[height=6.5em]{logo_cal.jpg}} % university/lab logo on left
{\bf Targeted Learning with the Moderated T-Statistic \vspace{0.5em}} % Poster title
{\textsc{\{ Nima Hejazi \& Andre Kurepa Waschka \} \hspace{12pt} Division of Biostatistics, UC Berkeley}}
  % Author names and institution
{\includegraphics[height=9em]{logo_sph.jpg}} % university/lab logo on right

%-------------------------------------------------------------------------------
% OVERVIEW
%-------------------------------------------------------------------------------

\headerbox{Overview}{name=overview,column=0,row=0}{

\begin{enumerate}\compresslist
\item This analysis seeks to generate optimal predictions for 5 genomic
  covariates across 67 sample observations, nearly evenly distributed across 5
  \textit{"knockout"} genotypes.
\item \textbf{Uninformative censoring:} the competition rules indicate that
  missingness was artificially introduced; this greatly simplifies the missing
  data problem to be solved via prediction and imputation methods.
\item Rather than use a single machine learning algorithm to estimate the
  missing values in the data set, a weighted combination of a library of
  learning algorithms is used to generate asymptotically optimal predictions.
\item While the theory underlying the Super Learner method is quite rich, at its
  core, the algorithm simply uses cross-validation to rank learning algorithms
  within a provided library according to a meta-learner, building a weighted
  combination of learning algorithms for prediction.
\end{enumerate}

\vspace{0.3em} % When there are two boxes, some whitespace may need to be added
               % if the one on the right has more content
}

%-------------------------------------------------------------------------------
% INTRODUCTION
%-------------------------------------------------------------------------------

\headerbox{Introduction}{name=introduction,column=1,row=0,bottomaligned=overview}{

\begin{itemize}
\item The goal of this prediction challenge is to infer the withheld values of a
  single genomic covariate for a subset of individuals from 5 randomly selected
  "knock-out" conditions in the full data set provided.
\item In order to predict the missing values in a \textit{provably optimal}
  manner, this analysis relies on the Super Learner algorithm, to generate
  asymptotically optimal prediction.
\item The problem of overfitting with the individual (and ensemble) learners is
  avoided by employing V-fold cross-validation (where $V = 10$ in the results
  presented).
\item The 5 genomic covariates that we provide predicted values for all have
  continuous measurements, thus, we use the squared error (L2) loss function
  with Super Learner.
\end{itemize}
}

%-------------------------------------------------------------------------------
% RESULTS 2
%-------------------------------------------------------------------------------

\headerbox{Results}{name=results,column=2,span=2,row=0}{

\begin{multicols}{2}

\begin{center}
\includegraphics[scale=0.25]{logo_cal}
\captionof{figure}{fitted vs true values for neutrophils}
\end{center}

\begin{center}
\includegraphics[scale=0.25]{logo_cal}
\captionof{figure}{fitted vs true values for lymphocytes}
\end{center}

\end{multicols}

%------------------------------------------------

\begin{multicols}{2}

\begin{center}
\includegraphics[scale=0.25]{logo_cal}
\captionof{figure}{fitted vs true values for monocytes}
\end{center}

\begin{center}
\includegraphics[scale=0.25]{logo_cal}
\captionof{figure}{fitted vs true values for basophils}
\end{center}

\end{multicols}
}

%-------------------------------------------------------------------------------
% REFERENCES
%-------------------------------------------------------------------------------

\headerbox{References}{name=references,column=2,above=bottom}{
\renewcommand{\section}[2]{\vskip 0.05em} % remove "References" section title
\nocite{*} % Insert publications even if they are not cited in the poster
\tiny{ % Reduce the font size in this block
\bibliographystyle{unsrt}
\bibliography{ph295_project}
}}

%-------------------------------------------------------------------------------
% Acknowledgements
%-------------------------------------------------------------------------------

\headerbox{Acknowledgements}{name=ack,column=3,aligned=references,above=bottom}{
% This block is as tall as the references block

We thank Prof. Alan E. Hubbard for his guidance and support in the completion of
this work.
}

%-------------------------------------------------------------------------------
% CONCLUSION
%-------------------------------------------------------------------------------

\headerbox{Conclusion}{name=conclusion,column=2,span=2,row=0,below=results,above=references}{

\begin{multicols}{2}

\begin{center}
\vspace*{-0.51cm}
\includegraphics[scale=0.25]{logo_cal}
\vspace{-1.8em}
\captionof{figure}{Graph from Correlation Structure}
\end{center}

%------------------------------------------------

\begin{itemize}
\item In order to visualize the relationship between the genomic covariates in
  the observed data set, a graph is generated from the correlation matrix.
\item We hold that a predictive analysis \textbf{\textit{does not target causal
  parameters}}. Thus, we refrain from providing a causal graph in our work.
\item Super Learner provides asymptotically optimal prediction, and our results
  display MSE values that substantiate this claim.
\end{itemize}

\end{multicols}
}

%-------------------------------------------------------------------------------
% METHODS
%-------------------------------------------------------------------------------

\headerbox{Methodology}{name=method,column=0, span=2,below=overview,bottomaligned=references}{
  % This block's bottom aligns with the bottom of the conclusion block

The \textbf{Super Learner} method works as follows:

\vspace{-0.2em}

\begin{itemize}\compresslist
\item Start by defining a base library of L learners:
  $\Psi^{1}, \dots, \Psi^{L}$ to be used within SuperLearner.
\item Specify a meta-learning method ($\Phi$), used to evaluate the base
  learners.
\item Use V-fold cross validation in each estimation step ($V = 10$ in our case)
  to protect against overfitting and evaluate learners.
\item Each base learner is used to generate fitted values for the training fold,
  generating a new matrix of subset-specific fits.
\item Then, the meta-learner is used to find the optimal combination of these
  fits.
\end{itemize}

In the analysis for this competition, we have used:

\vspace{-0.2em}

\begin{itemize}\compresslist
\item The full data set, iteratively predicting values for the 5 genomic
  covariates of interest.
\item In each run of Super Learner, indicator variables are used to impute the
  missing values remaining in the training set.
\end{itemize}
}

%-------------------------------------------------------------------------------

\end{poster}

\end{document}
